Gopkg.toml

```diff
@@ -1,10 +1,25 @@
 [[constraint]]
+  name = "github.com/googleapis/gnostic"
+  version = "v0.2.0"
+
+[[override]]
   name = "k8s.io/client-go"
-  version = "kubernetes-1.13.1"
+  # revision for tag "kubernetes-1.12.3"
+  revision = "d082d5923d3cc0bfbb066ee5fbdea3d0ca79acf8"
 
-[[constraint]]
+[[override]]
```

Comment:
Do these 3-4 need to be override, instead of constraint?

Reply:
dependencies have been cleaned up